src/Http/Requests/UpdateMasterProductVariant.php

```diff
@@ -31,7 +33,8 @@ public function rules()
             'excerpt' => 'sometimes|nullable|max:8192',
             'description' => 'sometimes|nullable|max:32768',
             'images' => 'nullable',
-            'images.*' => 'image|mimes:jpg,jpeg,pjpg,png,gif,webp'
+            'images.*' => 'image|mimes:jpg,jpeg,pjpg,png,gif,webp',
+            'state' => ['required', Rule::in(ProductStateProxy::values())],
```

Comment:
The field is nullable: https://github.com/vanilophp/framework/blob/4.1.0/src/MasterProduct/resources/database/migrations/2023_01_25_114755_add_description_to_variants_table.php#L14
the validation must match that logic